packages/blocks/src/components/slash-menu/slash-menu-node.ts

```diff
@@ -0,0 +1,238 @@
+import { getRichTextByModel } from '@blocksuite/blocks/std.js';
```

Comment:
don't import itself

Reply:
Blame the incorrect auto import!

Reply:
We could have a eslint rule to avoid this

Reply:
See https://github.com/toeverything/blocksuite/pull/762